packages/server/lib/browsers/chrome.ts

```diff
@@ -580,17 +570,17 @@ export = {
     await this._setAutomation(pageCriClient, automation, browserCriClient.resetBrowserTargets, options)
   },
 
-  async attachListeners (browser: Browser, url: string, pageCriClient, automation: Automation, options: BrowserLaunchOpts & { onInitializeNewBrowserTab?: () => void }) {
+  async attachListeners (browser: Browser, url: string, pageCriClient: CriClient, automation: Automation, options: BrowserLaunchOpts | BrowserNewTabOpts) {
     if (!browserCriClient) throw new Error('Missing browserCriClient in attachListeners')
 
-    await this._setAutomation(pageCriClient, automation, browserCriClient.resetBrowserTargets, options)
+    const cdpAutomation = await this._setAutomation(pageCriClient, automation, browserCriClient.resetBrowserTargets, options)
 
     await pageCriClient.send('Page.enable')
 
-    await options.onInitializeNewBrowserTab?.()
+    await options['onInitializeNewBrowserTab']?.()
```

Comment:
It bypasses type checks on `options`, otherwise you get: `Property 'onInitializeNewBrowserTab' does not exist on type 'BrowserLaunchOpts | BrowserNewTabOpts'.` since it's only on one of the two.

Open to suggestions on a cleaner way to do this... `await (options as BrowserNewTabOpts).onInitializeNewBrowserTab?.()` ??